src/ViewInstance.js

```diff
@@ -266,6 +275,7 @@ class ViewInstance extends React.Component {
 
     if ((isMARCSourceRecord || isLinkedDataSourceRecord) && (isViewingAnotherRecord || recordSourceWasChanged)) {
       this.getMARCRecord();
+      this.getLinkedDataEditorId();
```

Comment:
When source = LINKED_DATA, there is no need to call the API for getting the linkedDataId. In such case, the linked data ID will be present in the `instance.identifiers` section of the instance record. 

Api has to be called only when source=MARC.

Reply:
Added a guard for when the source is not MARC